util/check_tool_requirements.py

```diff
@@ -68,6 +68,34 @@ def get_verilator_version():
         return None
 
 
+def convert_verible_version(version_string):
+    '''
+    Convert Verible version string into a format compatible with
+    StrictVersion().
+    '''
```

Comment:
Nit: The standard python Docstring format is a sentence on a single line, then a blank line, then any supplementary information. Here, I'd probably try to squash everything into a single line. Maybe something like this?
```python
def convert_verible_version(version_string):
    '''Convert Verible version string to semantic versioning format.'''
```

Reply:
Ah, yes. This was meant to be just one sentence, but it got too long for a single line. Let me change that.